src/NodeJS/Server.php

```diff
@@ -254,6 +262,52 @@ public function getConnection()
     }
 
     /**
+     * Return the options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->options;
+    }
+
+    /**
+     * Set options array.
+     */
+    public function setOptions($options)
+    {
+        // pass these to the setOption() method which will validate.
+        foreach ($options as $key => $value) {
+            $this->setOption($key, $value);
+        }
+    }
+
+    /**
+     * Set a single option.
+     */
+    public function setOption($option, $value)
```

Comment:
Inline means:

1. put body the method (`setOption` in this case) inside the method, that calls it (`setOptions` in this case)
2. remove original method (`setOption` in this case)